core/src/main/kotlin/io/github/serpro69/kfaker/provider/BojackHorseman.kt

```diff
@@ -7,9 +7,9 @@ import io.github.serpro69.kfaker.dictionary.*
  * [FakeDataProvider] implementation for [CategoryName.BOJACK_HORSEMAN] category.
  */
 @Suppress("unused")
-class BojackHoreseman internal constructor(fakerService: FakerService) : AbstractFakeDataProvider<BojackHoreseman>(fakerService) {
+class BojackHorseman internal constructor(fakerService: FakerService) : AbstractFakeDataProvider<BojackHorseman>(fakerService) {
```

Comment:
Can you add a new class that will live along the old one instead of replacing it? And add @Deprecated to the old class?
It's fine to "replace" things with correct names in documentation, and in the cli-bot as well, but I would prefer not to introduce breaking changes to the "core" without deprecating the old class.